PySoar/exportClass.py

```diff
@@ -51,7 +51,7 @@ def initiate_best_worst(self, settings, no_legs):
     def __init__(self, settings, no_legs):
         self.file_name = settings.file_name
 
-        self.wb = xlwt.Workbook()  # initialize excel sheet
+        self.wb = xlwt.Workbook(encoding='latin-1')  # initialize excel sheet
```

Comment:
And latin was the first to got it working with but maybe it is better to use something else